src/woo_publications/publications/api/viewsets.py

```diff
@@ -384,3 +386,22 @@ def perform_update(self, serializer):
                 )
             case _:  # pragma: no cover
                 pass
+
+
+@extend_schema(tags=["Onderwerpen"])
+@extend_schema_view(
+    list=extend_schema(
+        summary=_("All available topics."),
+        description=_("Returns a paginated result list of existing topics."),
+    ),
+    retrieve=extend_schema(
+        summary=_("Retrieve a specific topic."),
+        description=_("Retrieve a specific topic."),
+    ),
+)
+class TopicViewSet(AuditTrailRetrieveMixin, viewsets.ReadOnlyModelViewSet):
+    queryset = Topic.objects.order_by("-registratiedatum")
```

Comment:
you are serializing a m2m relation - it's probably best to add that field to a `prefetch_related` or query performance will be really bad for the list endpoint